### PR TITLE
Intel AMX accelerated Medical Diagnosis.

### PR DESCRIPTION
**Pull request by pgrabusz**

### CODE REVIEW

File: charts/all/nfd/templates/nfd-namespaces.yaml
Comment (day0hero) [inline charts/all/nfd/templates/nfd-namespaces.yaml:1]: This should be removed. 

The correct location to declare namespaces is in values-hub.yaml (which you have already)

File: charts/all/nfd/templates/nfd-subscriptions.yaml
Comment (day0hero) [inline charts/all/nfd/templates/nfd-subscriptions.yaml:1]: This should be removed. 

The correct location to declare Subscriptions is in values-hub.yaml (which you already have)

File: charts/all/medical-diagnosis/database/templates/deploymentconfig.yaml [hunk 1]
```diff
@@ -14,7 +14,8 @@ spec:
     requests:
       storage: 1Gi
 ##storageClassName: {{ .Values.storageClassName }}
-  storageClassName: {{ .Values.global.datacenter.storageClassName }}
+##storageClassName: {{ .Values.global.datacenter.storageClassName }}
+  storageClassName: {{ .Values.database.storageClassName }}
```
Comment (beekhof) [inline charts/all/medical-diagnosis/database/templates/deploymentconfig.yaml:18]: Probably we should make use of the `default` directive here for compatibility

File: charts/all/medical-diagnosis/grafana/templates/xraylab-dashboard.yaml [hunk 2]
```diff
@@ -363,7 +363,7 @@ spec:
           "timeShift": null,
           "title": "Last uploaded image",
           "type": "ryantxu-ajax-panel",
-          "url": "https://image-server-xraylab-1.apps.{{ .Values.global.datacenter.clustername }}.{{ .Values.global.datacenter.domain }}/last_image_small/{{ .Values.global.xraylab.s3.bucketBaseName }}",
+          "url": "https://image-server-xraylab-1.apps.{{ .Values.global.clusterDomain }}/last_image_small/{{ .Values.global.xraylab.s3.bucketBaseName }}",
```
Comment (beekhof) [inline charts/all/medical-diagnosis/grafana/templates/xraylab-dashboard.yaml:366]: @day0hero why the non-standard DNS variables?  shouldn't this be coming from the operator's automatic variables?

File: charts/all/medical-diagnosis/image-generator/templates/image-generator-cm.yaml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+    secret_key = os.environ['AWS_SECRET_ACCESS_KEY']
+
+    service_point = os.environ['SERVICE_POINT']
+
+
+    print(service_point)
+
+
+    s3client = boto3.client('s3', 'us-east-1', endpoint_url=service_point,
```
Comment (beekhof) [inline charts/all/medical-diagnosis/image-generator/templates/image-generator-cm.yaml:53]: Pretty sure the region should be configurable

File: charts/all/medical-diagnosis/xray-init/templates/riskAssessment/kafkaservice.yaml [hunk 4]
```diff
@@ -14,7 +14,7 @@ spec:
           revisionTimestamp: ''
     spec:
       containers:
-      - image: quay.io/rh-data-services/xraylab-risk-assessment:latest
+      - image: quay.io/hybridcloudpatterns/hardware-accelerated-risk-assessment:latest
```
Comment (beekhof) [inline charts/all/medical-diagnosis/xray-init/templates/riskAssessment/kafkaservice.yaml:17]: @day0hero can we perhaps drive this from `value-hub.yaml` as an application override?
Maybe we should make a copy of it... `values-amx.yaml`

File: charts/all/openshift-data-foundations/values.yaml [hunk 5]
```diff
@@ -4,6 +4,7 @@ global:
   datacenter:
     storageClassName: gp2
     region: us-east-1
+  clusterDomain: gitops.com
```
Comment (beekhof) [inline charts/all/openshift-data-foundations/values.yaml:7]: Not loving all these DNS variables

**Pull request closed without merging**